src/components/Table/Table.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+
+import TableBody from './TableBody';
+import TableCell from './TableCell';
+import TableHead, { TableHeadBase } from './TableHead';
+import TableRow, { TableRowBase } from './TableRow';
+
+
+const Table = styled.table`
```

Comment:
could you add the typography styles here? do you know if it will make the font styling the same all across the table? 

I'm just thinking of having the typography styles stay the same if you do
```jsx
<Table>
   <tr>
      <th> Heading </th>
      <th>adfasdf</th>
    </tr>
     <tr>
         <td>.... 
```
instead of using your <Table.Cell> <Table.Row> components etc

Reply:
If the table declares a typography styles, they usually just get overridden by a browser's default stylesheet since it's common to style <th> and <td> directly. See my new change in Table.js to see if it's a cleaner approach to this issue!

Reply:
Ah yeah i like that approach